cartographi/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.0.5"
+__version__ = "1.1.0"
```

Comment:
There is a conflict with the base branch here that needs to be resolved before merging